src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -1530,9 +1551,11 @@ public enum DarknessAblaze implements LogicCardInfo {
         move "Risk Taker", {
           text "Flip a coin, if heads discard 5 cards from the top of your opponent’s deck. If tails, discard 5 cards from the top of your deck."
           energyCost C, C
-          attackRequirement {}
+          attackRequirement {
+            assert my.deck || opp.deck : "Both decks are empty"
+          }
           onAttack {
-
+            flip {if (opp.deck) opp.deck[0..4].discard()}, {if (my.deck) my.deck[0..4].discard()}
```

Comment:
I think you need to use `flip 1, {...}, {...}`

Reply:
`static flip (Closure eachHead, Closure eachTail={}, multi=[:])`
This overload should be the one this code ends up using, which is an implied 1 flip.